README.md

```diff
@@ -545,16 +555,6 @@ With pytest we can assert the logs we expect to be written by utilising pytest's
 
 Please refer to the specific packages documentation for more details. Details can be found in the [pytest logging documentation](https://docs.pytest.org/en/7.1.x/how-to/logging.html#how-to-manage-logging).
 
-### Asset management
-
-Custom CSS and JavaScript files are merged and compressed using [Flask Assets](https://flask-assets.readthedocs.io/en/latest/) and [Webassets](https://webassets.readthedocs.io/en/latest/). This takes all `*.css` files in `app/static/src/css` and all `*.js` files in `app/static/src/js` and outputs a single compressed file to both `app/static/dist/css` and `app/static/dist/js` respectively.
-
-CSS is [minified](https://en.wikipedia.org/wiki/Minification_(programming)) using [CSSMin](https://github.com/zacharyvoase/cssmin) and JavaScript is minified using [JSMin](https://github.com/tikitu/jsmin/). This removes all whitespace characters, comments and line breaks to reduce the size of the source code, making its transmission over a network more efficient.
-
-### Cache busting
-
-Merged and compressed assets are browser cache busted on update by modifying their URL with their MD5 hash using [Flask Assets](https://flask-assets.readthedocs.io/en/latest/) and [Webassets](https://webassets.readthedocs.io/en/latest/). The MD5 hash is appended to the file name, for example `custom-d41d8cd9.css` instead of a query string, to support certain older browsers and proxies that ignore the querystring in their caching behaviour.
```

Comment:
I know we no longer have this as removed flask assets and currently we just build main.css file from main.scss but this shouldnt affect us in our deployment due to running in lambda. 

We may even move our assets to cloudfront anyway so we'd be able to deal with that there if we decide to. @colinbowen thoughts?

Reply:
Yeah definitely we should look to having at least the error pages set up to be retrieved from CloudFront. Worth having a discussion with Ben / Rahat too

Reply:
I agree, that we should put static content in Cloudfront, having it cached will improve the performance too.